### PR TITLE
7 post comments

### PR DESCRIPTION
**Pull request by AydaFox**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -155,4 +155,64 @@ describe("/api/articles/:article_id/comments", () => {
...
+            article_id: 2
+        }
+        return request(app)
+            .post("/api/articles/2/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.comment).toMatchObject(expectedResponse);
+                expect(new Date(body.comment.created_at).toString()).toBe(timestamp);
+            });
```
Comment (johnhutcheon) [inline __tests__/app.test.js:249]: Could we get another test here for what would happen if we add unnecessary properties? I.e. if we add something other than username or body as well as those properties?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -155,4 +155,64 @@ describe("/api/articles/:article_id/comments", () => {
...
+            body: 'what a crazy comment I just left lol'
+        };
+        return request(app)
+            .post("/api/articles/99/comments")
+            .send(newComment)
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("article not found");
+            });
+    });
```
Comment (johnhutcheon) [inline __tests__/app.test.js:311]: Could we just add one more test for if the username does not exist please?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Nice work Ayda! I've just left some feedback requesting a couple of extra tests please 🦊

**Review by nickkagame — APPROVED** [review]
All good Ayda!

**Pull request merged**